package.json

```diff
@@ -23,11 +23,14 @@
     "url": "https://github.com/anyfin/bankid/issues"
   },
   "dependencies": {
+    "@types/node": "^12.11.5",
     "axios": "^0.18.1"
   },
   "devDependencies": {
     "@babel/cli": "^7.6.4",
     "@babel/core": "^7.6.4",
-    "@babel/preset-env": "^7.6.3"
+    "@babel/preset-env": "^7.6.3",
```

Comment:
If add typescript as a dev dependency I think we could drop all the @babel dependencies.